_includes/templates/nav-about-us-media.html

```diff
@@ -0,0 +1,11 @@
+<a href="/the-bureau/">
+    <img src="../../static/img/nav-about-us-video.png"
```

Comment:
Yes, we did intend to, though it probably deserves another conversation now that the design and UX team has shifted significantly.